bundles/action/org.openhab.action.pebble/README.md

```diff
@@ -1,16 +1,16 @@
 # Pebble Actions
 
-The Pebble action service allows you to send pins and notifications directly to your Pebble watch. 
+The Pebble action service allows you to send pins and notifications directly to your Pebble watch.
 
 You'll need to install the openHAB app on your Pebble watch and lookup the Timeline token. This token is displayed at the bottom of the configuration page when you configure the app on your phone.
 
 ## Configuration
 
 You can either configure the required Pebble token in `services/pebble.cfg` and refer to it by name, or provide it explicitly in your action calls.
 
-| Property | Default | Required | Description |
-|----------|---------|:--------:|-------------|
-| `<name>.token` | | | A token to use in action calls by name |
+| Property       | Default | Required | Description                            |
+|----------------|---------|-:-:------|----------------------------------------|

```

Comment:
This is broken.

Reply:
@9037568 I believe my latest commit fixes this.  Please confirm, and if it is okay, please merge.  Thanks and Happy New Year.

Reply:
Happy New Year to you!

However, I'm afraid the table is still broken.